src/TextInput.js

```diff
@@ -245,7 +239,11 @@ TextInput.propTypes = {
   /*
    * email type
    */
-  email: PropTypes.bool
+  email: PropTypes.bool,
+  /*
+   * dataLength type
+   */
+  dataLength: PropTypes.number
```

Comment:
let's remove this prop, we are only using `data-length` as a data attribute, but no need to document it here